ui/summit-2024/src/components/ConnectWalletModal/ConnectWalletModal.tsx

```diff
@@ -244,7 +244,7 @@ const ConnectWalletModal = (props: ConnectWalletProps) => {
         };
       case ConnectWalletFlow.CONNECT_IDENTITY_WALLET:
         return {
-          title: "Use IDW Wallet to connect",
+          title: "Use ID Wallet to connect",
```

Comment:
these also should all be in i18n - maybe we can just create a catch all ticket to convert these last minute if we have time.